babel.config.js

```diff
@@ -25,52 +25,218 @@
 
 'use strict';
 
-const minimist = require('minimist');
+const argv = require('minimist')(process.argv.slice(2));
+const experimentsConfig = require('./build-system/global-configs/experiments-config.json');
+const experimentsConstantBackup = require('./build-system/global-configs/experiments-const.json');
 const {isTravisBuild} = require('./build-system/common/travis');
-const argv = minimist(process.argv.slice(2));
-
-const isClosureCompiler =
-  argv._.includes('dist') || argv._.includes('check-types');
-const {esm} = argv;
-const noModuleTarget = {
-  'browsers': isTravisBuild()
-    ? ['Last 2 versions', 'safari >= 9']
-    : ['Last 2 versions'],
+
+const jsxOpts = {
```

Comment:
I think this refactor is a good opportunity to document each of these constants with a short comment. @jridgewell, @erwinmombay, or I can help with the ones that aren't obvious.